srcpkgs/incus/files/README.voidlinux

```diff
@@ -9,3 +9,15 @@ containers. For example, the command
 will assign a range of 65,536 IDs for users and groups.
 Some container configurations may require that the `CGROUP_MODE`
 variable in `/etc/rc.conf` be set to `unified`.
+
+
+Optional dependencies:
+
+- To run VMs install `qemu` `edk2-ovmf` and add (replace x64 as needed)
+
+	export INCUS_EDK2_PATH=/usr/share/edk2/x64
```

Comment:
why can't we just do this in the run file always? does it fail if the path doesn't exist?

also, that path seems x86_64-specific

Reply:
yes, it is (as noted)